src/__tests__/scenes/ChangePasswordScene.test.tsx

```diff
@@ -2,18 +2,23 @@ import { describe, expect, it } from '@jest/globals'
 import * as React from 'react'
 import { createRenderer } from 'react-test-renderer/shallow'
```

Comment:
Attempted to use `import TestRenderer from 'react-test-renderer'`, but errors for wrapped LoginUi scenes.

Reply:
Please create a P4 Asana task in 4.2.0 to fix this, then. We are probably importing some native library that isn't mocked properly. As it stands, the snapshots don't include *any* rendered output from the login-ui scenes, so we aren't really testing anything at this point.

Another way would be to put `it.skip`, so we just skip the test entirely, but still use the deep renderer. No sense running a useless test.